src/toPairs.ts

```diff
@@ -3,10 +3,37 @@
  * @param object
  * @signature
  *    R.toPairs(object)
+ *    R.toPairs.strict(object)
  * @example
  *    R.toPairs({ a: 1, b: 2, c: 3 }) // => [['a', 1], ['b', 2], ['c', 3]]
+ *    R.toPairs.strict({ a: 1 } as { a?: number }) // => [['a', 1]] typed Array<['a', number]>
```

Comment:
it looks a bit weird in the docs to have a type assertion. would this also work with `as const` ?

```suggestion
 *    R.toPairs.strict({ a: 1 } as const) // => [['a', 1]] typed Array<['a', number]>
```

Reply:
I got so fixated on the optionality aspect of things that I forgot that this is just a general example. I'll change it (the suggestion you provided has the wrong type for the values, it should be `1` and not `number`).